mico-core/src/main/java/io/github/ust/mico/core/resource/ApplicationResource.java

```diff
@@ -235,16 +239,16 @@
     public ResponseEntity<Resource<KFConnectorDeploymentInfoResponseDTO>> addKafkaFaasConnectorInstanceToApplication(@PathVariable(PATH_VARIABLE_SHORT_NAME) String applicationShortName,
                                                                                                                      @PathVariable(PATH_VARIABLE_VERSION) String applicationVersion,
                                                                                                                      @PathVariable(PATH_VARIABLE_KAFKA_FAAS_CONNECTOR_VERSION) String kfConnectorVersion) {
-        KFConnectorDeploymentInfo kfConnectorDeploymentInfo;
+        MicoServiceDeploymentInfo kafkaFaasConnectorSDI;
```

Comment:
Add issue to change delete mapping to get rid of the conflicting variable use in path. (conflict with post semantic)

Reply:
Issue already exists: #793